### PR TITLE
fixed clipped image drawing

### PR DESCRIPTION
**Pull request by mariotaku**
### Description of the feature or fix

A clear and concise description of what the bug or new feature is.

### Checkpoints
- [x] Follow the [styling guide](https://github.com/lvgl/lvgl/blob/master/docs/CODING_STYLE.md)
- [x] Run `code-format.py` from the `scripts` folder. [astyle](http://astyle.sourceforge.net/install.html) needs to be installed.
- [ ] Update the documentation

### CODE REVIEW

**Comment by mariotaku** [comment]
Could you review this fix? As the issue causes crashes sometime.

File: lv_conf_template.h [hunk 1]
```diff
@@ -176,6 +176,7 @@
 #if LV_USE_GPU_SDL
     #define LV_GPU_SDL_INCLUDE_PATH <SDL2/SDL.h>
     #define LV_GPU_SDL_LRU_SIZE (1024 * 1024 * 8)
+    #define LV_GPU_SDL_CUSTOM_BLEND_MODE (SDL_VERSION_ATLEAST(2, 0, 6))
```
Comment (kisvegabor) [inline lv_conf_template.h:181]: Please add some comment about the meaning of this and `LV_GPU_SDL_LRU_SIZE` too.

**Review by kisvegabor — COMMENTED** [review]
Looks good, only one thing.

**Comment by mariotaku** [comment]
@kisvegabor Done :)

**Comment by kisvegabor** [comment]
Thank you, merging!

**Pull request merged**